build/docs/build_docs.py

```diff
@@ -120,8 +120,10 @@ def format_as_document(magic, uid_base : str) -> MagicReferenceDocument:
     uid = f"{uid_base}.{safe_name}"
     metadata = {
         'title': f"{magic_name} (magic command)",
+        'author': 'rmshaffer',
```

Comment:
This is probably fine, but I wonder if it would be better to use some more general ID here. One option is to use `microsoftopensource`. I don't know if there is any service account that is more specific to the Quantum team. @cgranade, do you happen to be aware of anything?